api-models/course-feedback.yaml

```diff
@@ -0,0 +1,158 @@
+openapi: "3.0.0"
+info:
+  title: Course Feedback API
+  version: "1.0.0"
+  description: API for managing Course feedback with field selection and full data retrieval.
+
+paths:
+  /course-feedback:
+    get:
+      summary: Retrieve a list of all Course feedbacks
+      operationId: listCourseFeedback
+      parameters:
+        - name: fields
+          in: query
+          description: Comma-separated list of fields to include in the response
```

Comment:
```suggestion
        - name: only
          in: query
          description: Array of course IDs to fetch a subset of courses
```

Reply:
You should not be able to select which fields you want to select, you only should have the choice to select all or a subset